tools/make.py

```diff
@@ -1034,10 +1035,7 @@ def main(argv):
         # Temporarily copy optionals_root for building. They will be removed later.
         optionals_modules = []
         optional_files = []
-        try:
-            copy_optionals_for_building(optionals_modules,optional_files)
-        except:
-            pass
+        copy_optionals_for_building(optionals_modules,optional_files)
```

Comment:
Is this going to work with us having no optionals in ACEX? I think it will error out for us.